process/dataValidators/txValidator_test.go

```diff
@@ -252,16 +251,3 @@ func TestTxValidator_CheckTxValidityTxIsOkShouldReturnTrue(t *testing.T) {
 	result := txValidator.CheckTxValidity(txValidatorHandler)
 	assert.Nil(t, result)
 }
-
-//------- IsInterfaceNil
-
-func TestTxValidator_IsInterfaceNil(t *testing.T) {
-	t.Parallel()
-
-	accounts := getAccAdapter(0, big.NewInt(0))
-	shardCoordinator := createMockCoordinator("_", 0)
-	txValidator, _ := dataValidators.NewTxValidator(accounts, shardCoordinator, 100)
-	txValidator = nil
```

Comment:
please replace with declaration and leave the assert

Reply:
here i cannot replace with a declaration because NewTxValidator will return a private type ... so to fix the warning i will write this line of code " _ = txValidator"